plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt

```diff
@@ -61,6 +61,7 @@ open class DefaultPageCreator(
             link(it.name.orEmpty(), it.dri)
             group {
                 +buildSignature(it)
+                +ContentBreakLine(platformData.toSet())
```

Comment:
I'd make this a helper function in the content builder

Reply:
ok